doc/index.md

```diff
@@ -6,6 +6,21 @@ title: Entity Framework 6
 Npgsql has an Entity Framework 6 provider. You can use it by installing the
 [EntityFramework6.Npgsql](https://www.nuget.org/packages/EntityFramework6.Npgsql/) nuget.
 
+## Basic Usage ##
+Create an inheritor of DbConfiguration in the same assembly as your entity framework DbContext. Ensure that you configure provider services, a provider factory, a default connection factory as shown below:
```

Comment:
"Create an inheritor of DbConfiguration" should be rephrased. Something like "Define a class that inherits from `DbConfiguration`.

Note the inline code formatting for `DbConfiguration` and `DbContext`.

Reply:
resolved.